content/docs/1.0.0/tutorial/introduction.md

```diff
@@ -25,35 +25,30 @@ ioFog requires a Linux environment, however this Tutorial has everything already
 
 ---
 
-#### Install Docker
+#### Install Docker and Docker-Compose
 
 You can install the latest version of Docker with following command:
 
-```sh
+```bash
 curl -sSf https://get.docker.com/ | sh
 ```
 
-or
-
-Download Docker for [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/), [Debian](https://docs.docker.com/install/linux/docker-ce/debian/), [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/), or [CentOS](https://docs.docker.com/install/linux/docker-ce/centos/).
+Or download Docker manually for: [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/), [Debian](https://docs.docker.com/install/linux/docker-ce/debian/), [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/) or [CentOS](https://docs.docker.com/install/linux/docker-ce/centos/).
 
-Next we need to install Docker Compose. For the latest instructions [see the Docker Compose Install Guide](https://docs.docker.com/compose/install/#install-compose), however it should be similar to this:
-
-```sh
-# This has version 1.22.0 hardcoded, you probably
-# should change it to what ever the latest version is!
+Next we need to install Docker Compose. For the latest instructions [see the Docker Compose Install Guide](https://docs.docker.com/compose/install/#install-compose). Note the version to download. This tutorial requires docker-compose version `1.24`.
```

Comment:
For the latest instructions**,** see the Docker Compose Install Guide. ~Note the version to download.~ This tutorial requires docker-compose version `1.24` **so make sure to download the correct version**.